cmd/process-agent/app/check.go

```diff
@@ -153,7 +153,12 @@ func runCheck(cfg *config.AgentConfig, ch checks.Check) error {
 		return fmt.Errorf("collection error: %s", err)
 	}
 
-	time.Sleep(1 * time.Second)
+	if ch.Name() == checks.ProcessEvents.Name() {
+		log.Info("Listening for process_events during 10s before running the check")
```

Comment:
I've ended up changing a bit the log line here https://github.com/DataDog/datadog-agent/pull/12390/commits/2b962fa19b7d8f6cde4a18e387c7693a52d97592 since now we have a `wait` arg that is shared across all checks.